modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -121,219 +96,417 @@ Note that this applies to all variations of CSV files (see examples below for ot
+...
 ----
 
 .Query
 [source, cypher]
 ----
-LOAD CSV FROM 'file:///artists.csv' AS line
-CALL {
-  WITH line
-  CREATE (:Artist {name: line[1], year: toInteger(line[2])})
-} IN TRANSACTIONS OF 500 ROWS
+LOAD CSV FROM 'https://data.neo4j.com/importing-cypher/persons.csv' AS row
+MERGE (p:Person)
+SET
+p.tmdbId = toInteger(row.person_tmdbId),
+p.name = row.name,
+p.born = date(row.born)
 ----
 
 .Result
-[role="queryresult"]
+[source, role="queryresult"]
 ----
 +-------------------+
 | No data returned. |
 +-------------------+
-Nodes created: 4
-Properties set: 8
-Labels added: 4
-Transactions committed: 1
+Nodes created: 868
+Properties set: 2604
+Labels added: 868
 ----
+====
 
+For a list of type casting functions, see xref:values-and-types/casting-data.adoc[Casting data values].
 
-[[load-csv-import-data-containing-escaped-characters]]
-== Import data containing escaped characters
-In this example, we both have additional quotes around the values, as well as escaped quotes inside one value.
 
-.artists-with-escaped-char.csv
-[source, csv, role="noheaders", filename="artists-with-escaped-char.csv"]
+=== Handle `null` values
+
+Neo4j does not store `null` values.
+`null` or empty fields in a CSV files can be skipped or replaced with default values in `LOAD CSV`.
+
+.Processing a file with `null` values
+====
+In the file `companies.csv`, some rows do not specify values for some columns.
+The examples show several options of treating `null` values.
+
+.companies.csv
+[source, csv, filename=companies.csv]
 ----
-"1","The ""Symbol""","1992"
+Id,Name,Location,Email,BusinessType
+1,Neo4j,San Mateo,contact@neo4j.com,P
+2,AAA,,info@aaa.com,
+3,BBB,Chicago, ,G
+,CCC,Michigan,info@ccc.com,G
 ----
 
-.Query
+.Skip `null` values
 [source, cypher]
 ----
-LOAD CSV FROM 'file:///artists-with-escaped-char.csv' AS line
-CREATE (a:Artist {name: line[1], year: toInteger(line[2])})
-RETURN
-  a.name AS name,
-  a.year AS year,
-  size(a.name) AS size
+LOAD CSV WITH HEADERS FROM 'file:///companies.csv' AS row
+WITH row WHERE row.Id IS NOT NULL
+MERGE (c:Company {id: row.Id});
+----
+
+.Provide a default for `null` values
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///companies.csv' AS row
+WITH row WHERE row.Id IS NOT NULL
+MERGE (c:Company {id: row.Id, hqLocation: coalesce(row.Location, "Unknown")})
+----
+
+.Change empty strings to `null` values (not stored)
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///companies.csv' AS row
+WITH row WHERE row.Id IS NOT NULL
+MERGE (c:Company {id: row.Id})
+SET c.email = nullIf(trim(row.Email), "")
+----
+====
+
+[TIP]
+Because `null` values are not stored in the database, a strategy for selectively getting rid of some values, for example the empty strings from the last query, is to map them into `null` values.
```

Comment:
This sentence is quite difficult to follow?

Reply:
rephrased